### PR TITLE
COMMERCE-5204 + COMMERCE-5216 remove CPDefinition/CPInstance dpendencies from shipping options

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderItemServiceImpl.java [hunk 1]
```diff
@@ -463,27 +463,26 @@ public CommerceOrderItem updateCommerceOrderItemPrices(
...
-			BigDecimal promoPriceWithTaxAmount,
-			BigDecimal discountAmountWithTaxAmount,
-			BigDecimal finalPriceWithTaxAmount,
 			BigDecimal discountPercentageLevel1WithTaxAmount,
 			BigDecimal discountPercentageLevel2WithTaxAmount,
 			BigDecimal discountPercentageLevel3WithTaxAmount,
-			BigDecimal discountPercentageLevel4WithTaxAmount)
+			BigDecimal discountPercentageLevel4WithTaxAmount,
+			BigDecimal promoPrice, BigDecimal promoPriceWithTaxAmount,
+			BigDecimal unitPrice, BigDecimal unitPriceWithTaxAmount)
```
Comment (dsitu) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderItemServiceImpl.java:485]: sorting looks off here

**Review by dsitu — CHANGES_REQUESTED** [review]
@luca-pellizzon for commit https://github.com/liferay-commerce/liferay-portal/pull/521/commits/5f8566bf96aa8f31fbc160c39be6fbc8ac4ee086, can we separate out the regen files from the manual changes? thanks.

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Review by marco-leo — APPROVED** [review]

**Review by riccardo-alberti — APPROVED** [review]

**Comment by riccardo-alberti** [comment]
ci:forward

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

**Comment by riccardo-alberti** [comment]
ci:forward

**Pull request closed without merging**